xwiki-platform-core/xwiki-platform-legacy/xwiki-platform-legacy-extension-handler-xar/src/main/java/org/xwiki/extension/xar/job/diff/DocumentVersionReference.java

```diff
@@ -28,7 +28,9 @@
  * 
  * @version $Id$
  * @since 7.0RC1
+ * @deprecated Use {@link org.xwiki.model.reference.DocumentVersionReference} instead.
  */
+@Deprecated(since = "14.8RC1")
 public class DocumentVersionReference extends DocumentReference
```

Comment:
Extending `org.xwiki.model.reference.DocumentVersionReference` might make retro compatibility easier. Also make DocumentUnifiedDiff.new implementation a bit simpler.

Reply:
Might avoid a bit of code duplication.